csharp/test/Ice/admin/AllTests.cs

```diff
@@ -486,11 +514,24 @@ public static void allTests(global::Test.TestHelper helper)
                     props.Add("Ice.Admin.Facets", "TestFacet");
                     var com = factory.createCommunicator(props);
                     Ice.ObjectPrx obj = com.getAdmin();
-                    Ice.PropertiesAdminPrx pa = Ice.PropertiesAdminPrxHelper.checkedCast(obj, "Properties");
-                    test(pa == null);
-                    Ice.ProcessPrx proc = Ice.ProcessPrxHelper.checkedCast(obj, "Process");
-                    test(proc == null);
-                    com.destroy();
```

Comment:
You removed a (seemingly) unrelated `com.destroy();` call here.

Reply:
Good catch. This was accidental. Fixed.